podium/talks/templates/talks/session.html

```diff
@@ -0,0 +1,59 @@
+{% extends "base.html" %}
+
+
+{% block content %}
+<div class="container">
+		<div class="row ">
+			<div class="
+				col-xs-12 
+				col-sm-10 col-sm-push-1
+				col-md-8 col-md-push-2
+				col-lg-8 col-lg-push-2
+				" > 
```

Comment:
This template uses Bootstrap-based CSS classes, but neither this file nor the base.html has been updated to include the required links to the Bootstrap files.  Please add the required includes to the HEAD tag in base.html via either including the files in the project or pointing to a public CDN copy of them.